mindsdb/integrations/handlers/chromadb_handler/chromadb_handler.py

```diff
@@ -276,14 +276,16 @@ def insert(
         """
         Insert data into the ChromaDB database.
         """
+
         collection = self._client.get_collection(table_name)
-        embeddings = data[TableField.EMBEDDINGS.value].tolist()
-        ids = data[TableField.ID.value].tolist()
-        documents = data[TableField.CONTENT.value].tolist()
-        metadata = data[TableField.METADATA.value].tolist()
+
+        data = data.to_dict(orient="list")
 
         collection.add(
-            ids=ids, documents=documents, embeddings=embeddings, metadatas=metadata
+            ids=data[TableField.ID.value],
+            documents=data.get(TableField.CONTENT.value),
+            embeddings=data[TableField.EMBEDDINGS.value],
+            metadatas=data.get(TableField.METADATA.value),
         )
```

Comment:
it will fail if we pass in a list containing None, if we pass in a single None it is fine